.github/workflows/apple-perf.yml

```diff
@@ -306,3 +306,78 @@ jobs:
       ios-xctestrun-zip: https://gha-artifacts.s3.amazonaws.com/${{ github.repository }}/${{ github.run_id }}/artifacts/Benchmark.xctestrun.zip
       test-spec: ${{ inputs.test_spec || 'https://ossci-ios.s3.amazonaws.com/executorch/default-ios-device-farm-appium-test-spec.yml' }}
       extra-data: https://gha-artifacts.s3.amazonaws.com/${{ github.repository }}/${{ github.run_id }}/artifacts/${{ matrix.model }}_${{ matrix.delegate }}/model.zip
+
```

Comment:
@huydhn Do you mind adding mps delegate to the scheduled workflow in this PR? 

CRON_DEFAULT_DELEGATES: "xnnpack,coreml" -> CRON_DEFAULT_DELEGATES: "xnnpack,coreml,mps"

Reply:
Sure, let me add that